### PR TITLE
Updated left out warning statement

### PR DESCRIPTION
**Pull request by daminichopra**
## Proposed changes

Fixed some left out "warning" statement as "warn" is deprecated now 

## Types of changes

What types of changes does your code introduce to the project: _Put
an `x` in the boxes that apply_

- [ x ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing
      functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after
creating the PR. If you're unsure about any of them, don't hesitate to
ask. We're here to help! This is simply a reminder of what we are going
to look for before merging your code._

- [ x ] I have read the [CONTRIBUTING][contrib] doc
- [ x ] I have signed the [CLA][cla]
- [ ] I have added tests that prove my fix is effective or that my
      feature works
- [ x ] I have updated the [RELEASE_NOTES][RELEASE_NOTES.md]
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in
      downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion
by explaining why you chose the solution you did and what alternatives
you considered, etc...

[cla]: https://raw.githubusercontent.com/orchestracities/ngsi-timeseries-api/master/individual_cla.pdf
    "Martel Open Source Software Individual Contributor License Agreement"
[contrib]: https://github.com/smartsdk/ngsi-timeseries-api/blob/master/CONTRIBUTING.md
    "Contributing to QuantumLeap"

### CODE REVIEW

File: src/reporter/op.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def query():
         if entities:
             if len(entities) > 1:
                 import warnings
```
Comment (chicco785) [inline src/reporter/op.py:67]: most probably then you can remove this import

File: src/translators/sql_translator.py [hunk 2]
```diff
@@ -267,7 +267,7 @@ def _insert_entities_of_type(self,
                 import warnings
```
Comment (chicco785) [inline src/translators/sql_translator.py:267]: most probably then you can remove this import

File: src/reporter/query_1T1ENA.py [hunk 3]
```diff
@@ -87,8 +87,7 @@ def query_1T1ENA(entity_id,   # In Path
 
     if entities:
         if len(entities) > 1:
-            import warnings
-            warnings.warn("Not expecting more than one result for a 1T1ENA.")
+            warnings.warning("Not expecting more than one result for a 1T1ENA.")
```
Comment (chicco785) [inline src/reporter/query_1T1ENA.py:90]: logging.warning ?

**Review by chicco785 — CHANGES_REQUESTED** [review]

File: src/reporter/query_1T1ENA.py (hunk 3 above)
Comment (daminichopra) [reply src/reporter/query_1T1ENA.py:90]: yes looking into it some issue in branch

**Review by chicco785 — APPROVED** [review]

**Comment by daminichopra** [comment]
@chicco785 , Gentle Reminder for PR merge

**Comment by chicco785** [comment]
> @chicco785 , Gentle Reminder for PR merge

hi, we move to 0.9.0-dev, i need you to rebase and append correctly your content to the release note, many thanks.

**Comment by c0c0n3** [comment]
@daminichopra I'd imagine we can close #606 after merging this PR?

**Comment by daminichopra** [comment]
> @daminichopra I'd imagine we can close #606 after merging this PR?

@c0c0n3 yes Issue #606 can be closed after merging this PR.

**Pull request merged**